components/ruby/puppet/puppet-common.p5m

```diff
@@ -34,131 +34,133 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 set name=org.opensolaris.smf.fmri value=svc:/application/puppet:agent \
     value=svc:/application/puppet:master
-dir  path=etc/puppet owner=puppet group=puppet mode=0755
+dir  path=etc/puppetlabs/ owner=puppet group=puppet mode=0755
+dir  path=etc/puppetlabs/code owner=puppet group=puppet mode=0755
+dir  path=etc/puppetlabs/puppet owner=puppet group=puppet mode=0755
 dir  path=var/lib/puppet owner=puppet group=puppet mode=0755
 dir  path=var/log/puppet owner=puppet group=puppet mode=0755
-file path=etc/puppet/auth.conf owner=puppet group=bin mode=0444
+file path=etc/puppetlabs/puppet/auth.conf owner=puppet group=bin mode=0444
```

Comment:
mode=0644 preseve=true?
The same  - for other etc/puppetlabs/*.conf files....

Reply:
Ah yep I have added that in the latest revision. I thought we had a general transform for etc.